app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+    </style>
+
+    <style name="button_style">
+        <item name="android:layout_marginBottom">@dimen/margin_between_button_xsmall</item>
+    </style>
+
+    <style name="icon_style">
+        <item name="android:layout_marginStart">@dimen/margin_start_icon_small</item>
```

Comment:
as this style is used only by one view it is not necessary, please remove it

Reply:
Solved!